framework/include/timesteppers/TimeStepper.h

```diff
@@ -138,6 +138,9 @@ class TimeStepper : public MooseObject, public Restartable, public ScalarCouplea
   /// Whether or not the previous solve converged.
   bool _converged;
 
+  /// Cutback factor if a time step fails to converge
+  const Real & _cutback_factor_at_failure;
```

Comment:
```suggestion
  const Real _cutback_factor_at_failure;
```

This should be captured by value.

Reply:
Well that depends. If you ever want to control this parameter from the Control system, capturing by reference is the correct way.

Reply:
I left it as-is.

Reply:
That would require a `params.declareControllable("cutback_factor_at_failure");`. Either add that, or get rid of the reference.